core/html_api.php

```diff
@@ -777,6 +775,8 @@ function html_body_end() {
 
 	echo '</div>', "\n";
 
+	event_signal( 'EVENT_LAYOUT_BODY_END' );
```

Comment:
As [suggested by @vboctor](https://www.mantisbt.org/bugs/view.php?id=20084#c51400), add comment explaining the significance of the position